Sources/PathKit.swift

```diff
@@ -152,8 +152,18 @@ extension Path {
   ///
   public func abbreviate() -> Path {
 #if os(Linux)
-    // TODO: actually de-normalize the path
-    return self
+    let env = ProcessInfo.processInfo.environment
+    guard let home = env["HOME"], !home.isEmpty else { return self }
+    guard let homeRange = self.path.range(of: home, options: [.anchored, .caseInsensitive]) else { return self }
+    let withoutHome = Path(self.path.replacingCharacters(in: homeRange, with: ""))
+    
+    if withoutHome.path.isEmpty || withoutHome.path == "/" {
```

Comment:
Note, `Path.separator` doesn't change at the moment, but that's a separate issue.

I'd probably reverse the `#if os()` too, so that this code will be used on freeBSD, windows etc. Could be `#if os(osx) || os(iOS) .. etc` with all Apple platforms. Or perhaps we should just use the same implementation everywhere so for consistency. What do you think?